src/test/rgw/CMakeLists.txt

```diff
@@ -63,3 +63,29 @@ target_link_libraries(ceph_test_rgw_obj
 set_target_properties(ceph_test_rgw_obj PROPERTIES COMPILE_FLAGS
   ${UNITTEST_CXX_FLAGS})
 
+# ceph_test_rgw_bencode
+set(test_rgw_bencode_srcs test_rgw_bencode.cc)
+add_executable(ceph_test_rgw_bencode
+  ${test_rgw_bencode_srcs}
+  )
+target_link_libraries(ceph_test_rgw_bencode
+  rgw_a
+  cls_rgw_client
+  cls_lock_client
+  cls_refcount_client
+  cls_log_client
+  cls_statelog_client
+  cls_version_client
+  cls_replica_log_client
+  cls_user_client
+  librados
+  global
+  curl
+  uuid
+  expat
+  ${CMAKE_DL_LIBS}
+  ${UNITTEST_LIBS}
+  ${CRYPTO_LIBS}
+  )
+set_target_properties(ceph_test_rgw_bencode PROPERTIES COMPILE_FLAGS
+  ${UNITTEST_CXX_FLAGS})
```

Comment:
this is all you should need here:
```cmake
add_executable(unittest_rgw_bencode test_rgw_bencode.cc)
add_ceph_unittest(unittest_rgw_bencode ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/unittest_rgw_bencode)
target_link_libraries(unittest_rgw_bencode rgw_a)
```
the `add_ceph_unittest()` part is what adds it to make check